apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -176,6 +161,12 @@ def _copy_standalone_crt(self, source_dir, standalone_crt_dir):
         "src/runtime/crt/tab",
     ]
 
+    FQBN_TOKEN = "<FQBN>"
```

Comment:
What are these tokens going to be used for? Would love a comment.

I would also prefer to have them defined as constants directly above where `generate_project` is located, since they are first referenced there.